spec/requests/forms_spec.rb

```diff
@@ -171,4 +135,39 @@
       end
     end
   end
+
+  describe "#index - Viewing all forms for an organisation" do
+    describe "Given a form" do
```

Comment:
This doesn't seem quite right as a description... do we even need the extra describe block here? Could we just have the test directly in the `#index ...` group?

Reply:
Fixed this now